modules/apps/collaboration/social/social-activities-web/src/main/resources/content/Language.properties

```diff
@@ -1,3 +1,4 @@
 javax.portlet.title.com_liferay_social_activities_web_portlet_SocialActivitiesPortlet=Activities
 maximum-activities-to-load-at-once=Maximum Activities to Load at Once
-see-more=See More
\ No newline at end of file
+see-more=See More
+following=Following
```

Comment:
The new lang key should be in alphabetic order. You should also run gradlew buildLang in this bundle in order to generate the keys (at least as automatic copies)